db/migrate/20231013123032_add_tags_to_submission_analyses.rb

```diff
@@ -0,0 +1,7 @@
+class AddTagsToSubmissionAnalyses < ActiveRecord::Migration[7.0]
+  def change
+    return if Rails.env.production?
+
+    add_column :submission_analyses, :tags_data, :text, null: true
```

Comment:
I've named it `tags_data` for:

a) consistency
b) to allow us to have a `tags` method to get the actual tags